libraries/provision/ansible/playbooks/install-sync-gateway-package.yml

```diff
@@ -17,8 +17,8 @@
   any_errors_fatal: true
   vars:
     # Primary node
-    couchbase_server_primary_node: "{{ hostvars[groups.couchbase_servers[0]].ansible_host }}"
-
+    # couchbase_server_primary_node: "{{ hostvars[groups.couchbase_servers[0]].ansible_host }}"
+    couchbase_server_primary_node: ""
```

Comment:
You are setting the value of couchbase_server_primary_node to "". Does it get replaced with the right set of IPs in your runs? As far as I know, it should be couchbase_server_primary_node:

Reply:
Yes, it will set it on python script and pass it on here. It is same as schema and port where it passes from python. I tested and made sure that it passes by checking the console.